src/errorhandling/index.ts

```diff
@@ -3,4 +3,5 @@
  * @ignore
  */
 
+export * as ErrorHandler from './ErrorHandler'
 export * from './ExtrinsicError'
```

Comment:
Shouldn't we handle it as the following `export * as ExtrinsicError from './ExtrinsicError'` too.

Reply:
mhhh I that's code I don't need to mess with, so I'd refrain from doing so and possibly break stuff